src/Symfony/Component/HttpClient/Response/MockResponse.php

```diff
@@ -43,9 +42,10 @@ class MockResponse implements ResponseInterface, StreamableInterface
      *
      * @see ResponseInterface::getInfo() for possible info, e.g. "response_headers"
      */
-    public function __construct(string|iterable $body = '', array $info = [])
-    {
-        $this->body = $body;
+    public function __construct(
+        private null|string|iterable $body = '',
```

Comment:
I would revert this change. The possibility for `$body` to be `null` is purely internal. I guess we shouldn't allow consumers of this class to pass `null` here.

Reply:
Indeed